src/utils/translations/pt.ts

```diff
@@ -0,0 +1,94 @@
+const pt = {
+  // Home Tab
+  Home: "Início",
+  "Period in": "Período em",
+  "no info": "desconhecido",
+  Period: "Período",
+  today: "hoje",
+  Delay: "Atraso",
+  Days: "(0)[Dias];(1)[Dia];(2-4)[Dias];(5-20)[Dias];(21)[Dia];(22-24)[Dias];(25-30)[Dias];(31)[Dia];(32-34)[Dias];(35-40)[Dias];",
```

Comment:
Here you copied days formatting from the Russian language where we use different rules for the number of days than other language groups, as example we write:
- 1 day
- 21 day
In Portuguese you should write:
- 1 dia
- 2 dias
like in English